### ISSUE TITLE
My changes are not showing after ninja build

### ISSUE THREAD

**Issue by XtreemSolution** [brave/browser-android-tabs#2038]
I have downloaded complete code and all dependencies. I'm able to make build in src/Out/Default folder using following commands:

`ninja -C out/Default chrome_public_apk`

`build/android/adb_install_apk.py out/Default/apks/Brave.apk`

I did a lot of changes in /src/Brave and run above 2 commands again.  But now the new apk is still same and original, my changes are not showing in the new build. I have tested it by replacing many text string but didn't see any changes in new build. 

I also deleted all files of /src/Out and make a new build but same.

So please help me how to make the new development apk build with my changes. 

Thanks

**Comment by XtreemSolution** [comment]
@deeppandya Could you help me?

**Comment by deeppandya** [comment]
@XtreemSolution when you deleted **/src/out**, did you run **sh scripts/getThirdParties.js** to initialise the build ?

**Comment by XtreemSolution** [comment]
> @XtreemSolution when you deleted **/src/out**, did you run **sh scripts/getThirdParties.js** to initialise the build ?

Not yet.. let me try.. thanks man for your reply :)

**Comment by XtreemSolution** [comment]
@deeppandya Brother I did following steps:

- Do all my changes in /src/brave

- Deleted all content of /src/out

- Run `gn args out/Default` and generate argument file again

- Run `sh scripts/getThirdParties.js`

- Execute `ninja -C out/Default chrome_public_apk`

- Do export using `build/android/adb_install_apk.py out/Default/apks/Brave.apk`

But still my phone installed the origin Brave browser without my changes. 

I'm not sure what I'm missing. Can you help me here?

**Comment by SergeyZhukovsky** [comment]
@XtreemSolution you deployed a debug version of app, it is called `Brave-Debug` and has a grey lion icon. Both official play market and debug apps can be installed together.

**Comment by XtreemSolution** [comment]
> @XtreemSolution you deployed a debug version of an app, it is called `Brave-Debug` and has a grey lion icon. Both official play market and debug apps can be installed together.

@SergeyZhukovsky  Correct!! ind if I make `is_debug = false` then the build version will be live version, that's fine. But I don't know why my changes are not reflecting in any build. I follow the procedure as I mentioned in my previous message.  https://github.com/brave/browser-android-tabs/issues/2038#issuecomment-533145250

**Comment by XtreemSolution** [comment]
@SergeyZhukovsky @deeppandya Guys please help to kickstart my build

**Comment by SergeyZhukovsky** [comment]
I never encountered such a problem. The changes are always reflected when I change them. Try to clean your `out` folder and build from scratch is only one thing that comes into my head.

**Comment by deeppandya** [comment]
@XtreemSolution if you have a branch with your changes, i can take a look.
